views/view_package.dt

```diff
@@ -72,6 +72,23 @@ block body
 				input(id="package-clipboard",value='dub add #{packageName}')
 				button(class="btn-clipboard",data-clipboard-target="#package-clipboard")
 					img(src="/images/clippy.svg", width=13, alt="Copy to clipboard")
+		div
```

Comment:
empty useless diff

Reply:
well, it groups this thingy, and it's what I applied the margins (top+bottom) to

Reply:
no you are missing an indentation level if you want this to group anything

Reply:
ah, I see…
just writing a class without any element type doesn't make it apply to the "current" one but automagically creates a new `div`